src/tools/pub/cmd/pub-global.md

```diff
@@ -110,8 +109,43 @@ and passes in two arguments.
 $ pub global run foo:bar arg1 arg2
```

Comment:
Above it says "use `run`". It'd be easy to think that THAT's the way to run it. But you'd normally wouldn't, you'd use pub global activate and then just run the executable like it was native.

I think this page needs a good example at the top. Like maybe show how to get Stagehand and run it (with the assumption that your path is set up). That way they have a goal to aim for, and they can return here to remind themselves how to run and update apps. This page would have all the nitty gritty details (including as much about path setting as you can stand to present), but it would start out simply.